rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -488,28 +496,53 @@
       "title": "Supplementary content",
       "description": "Textual description of supplementary contents (special issues, etc.).",
       "type": "string",
-      "minLength": 1
+      "minLength": 1,
+      "form": {
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
     },
     "index": {
       "title": "Indexes",
       "description": "Textual description of the indexes.",
       "type": "string",
-      "minLength": 1
+      "minLength": 1,
+      "form": {
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
     },
     "missing_issues": {
       "title": "Missing issues",
       "description": "Textual description of the missing issues.",
       "type": "string",
-      "minLength": 1
+      "minLength": 1,
+      "form": {
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
     },
     "issue_binding": {
       "title": "Number of issues to be bound together",
       "type": "number",
-      "minimum": 0
+      "minimum": 0,
+      "form": {
+        "hide": true,
```

Comment:
Don't understand why we have to hide this properties?

Reply:
When creating a new holdings, only few fields are available by default. For the other hidden fields , the librarian needs to choose them manually. I added screen capture to the PR to show this